pom.xml

```diff
@@ -14,141 +14,101 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
-
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <groupId>org.wso2</groupId>
         <artifactId>wso2</artifactId>
         <version>5</version>
     </parent>
-
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>org.wso2.carbon.securevault</artifactId>
-    <version>5.2.0-SNAPSHOT</version>
-    <packaging>bundle</packaging>
-    <name>WSO2 Carbon Kernel - Secure Vault</name>
-    <description>
-        Secure vault bundle
-    </description>
+    <groupId>org.wso2.carbon</groupId>
```

Comment:
this is carbon kernel groupId and should not be used here since it is a separate repo. You need to change this.